### PR TITLE
Jtmalarecki add map to index

### PR DESCRIPTION
**Pull request by jtmalarecki**
Added Google Map component. Added it to the index page.

### CODE REVIEW

File: components/GoogleMap.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export default {
+  data: function () {
+    return {
+      map: false
+    }
+  },
+  mounted() {
+    // TODO: A reference error will occur if scriptjs is define above and not here
+    scriptjs = require('scriptjs')
```
Comment (JeremiahPinto) [inline components/GoogleMap.vue]: Remove the `let` above and declare as `const` here in mounted

File: components/GoogleMap.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    // Load the Google Maps API
+    scriptjs('https://maps.googleapis.com/maps/api/js?key=' + process.env.googleMapsApi, () => {
+      this.initMap()
+    })
+  },
+  methods: {
+    // The callback after the Google Maps JS API loads
+    initMap: () => {
```
Comment (JeremiahPinto) [inline components/GoogleMap.vue]: Declaring arrow functions in methods can cause issues, just use the short hand notation i.e.
`initMap() {`

File: layouts/default.vue [hunk 3]
```diff
@@ -58,7 +58,7 @@
       </v-btn>
     </v-toolbar>
     <v-content>
-      <v-container>
+      <v-container fill-height>
```
Comment (JeremiahPinto) [inline layouts/default.vue]: Add `class="pa-0 ma-0"` to the container to fill the page with just the map

**Review by JeremiahPinto — CHANGES_REQUESTED** [review]
Check out the single-line comments I made in the files, otherwise it was pretty good.

File: layouts/default.vue (hunk 3 above)
Comment (jtmalarecki) [reply layouts/default.vue]: When I add class="pa-0 ma-0" the container collapses to 0 height. I've been using height=100% and width=100% for the map, but I think these take 100% of 0. I'm not sure how to get the map to fill the available area. I'm thinking with a flexbox and the grow property, but I haven't yet experimented with this.

Comment (jtmalarecki) [reply layouts/default.vue]: Scratch that, I was replacing fill-height. However, the v-container does not fill the width of the screen on desktop as it's the max-width property is set to 900px.

**Review by JeremiahPinto — APPROVED** [review]

**Pull request merged**